### PR TITLE
50154_sign_in_sign_out

### PR DESCRIPTION
**Pull request by mdmahmudurrahman**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div class="center jumbotron">
   <h1><%= t "welcome_message" %> <br> <%= t "system_name" %></h1>
-
-  <%= link_to t(:sign_up_now), new_user_registration_path, class: "btn btn-lg btn-primary" %>
+  <% if !user_signed_in? %>
```
Comment (MuhammadTamzid) [inline app/views/static_pages/home.html.erb]: <% unless user_signed_in? %>

**Comment by MuhammadTamzid** [comment]
OK

**Comment by sujoydatta** [comment]
ok

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -4,6 +4,24 @@
     <%= link_to t(:system_name), root_path, id: "logo" %>
     <nav>
       <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "About", about_path%></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -4,6 +4,24 @@
...
+        <li><%= link_to "Users", "#" %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

**Pull request merged**